brainlit/utils/session.py

```diff
@@ -135,6 +136,7 @@ def get_segments(self, seg_id: int, bbox: Optional[Bounds] = None) -> nx.Graph:
             G = get_sub_neuron(G, [bbox[:3], bbox[3:]])
         return G
 
+    # TOMMY REVIEW
     def create_tubes(
```

Comment:
something more like "Create voxel-wise foreground/background labels associated with a particular neuron trace within a given bounding box of voxel coordinates."

Reply:
done